docs/platform/release-notes/tizen-6-5-m2.md

```diff
@@ -0,0 +1,400 @@
+# Tizen 6.5 Public M2
+
+Release date: Oct. 31, 2021
+
+
+## Release Notes
+
+### System (Kernel and System framework)
+
+#### New and changed features
+
+- New API and features
+  - Kernel
+    - A fine-grained version of THP (Transparent HugePage) in Linux kernel for embedded systems has been implemented in Raspberry Pi 4.
+  - HAL
+    - HAL API Layer is implemented for unifying the HAL backend style and splitting out the hardware dependency.
+    - New image structure with hal.img is adapted for HAL, a set of HAL backend set in /hal.
+  - System and resource management
+    - VIP process monitor has been improved by removing race conditions.
+    - Percentage-based memory threshold has been developed.
+    - Coredump generation for .NET 6 runtime has been supported.
+    - Tizen feature database tools have been improved.
+    - Container adaptation for file system mount units has been developed.
+  - Device and sensor management
+    - Orientation and rotation vector sensor algorithms have been upgraded for improving correctness.
+    - HAL backend for Inertial Measurement Unit (IMU) sensors has been implemented for Raspberry Pi 4 devices.
+    - Separate backend libraries for sensor devices have been supported.
+    - Daemon-less direct API set for peripheral-io and runtime memory usage has been developed.
+    - Sleep mode for IoT headless devices has been supported.
+    - Display control for HDMI-based external display devices has been supported.
+    - Display option for USB connection change has been added.
+    - API for board serial number has been added.
+  - OS upgrade
+    - FOTA update for Raspberry Pi 4 IoT headless devices has been supported.
+    - Crash recovery schemes have been developed for reliable OS upgrade.
+    - Metadata for FOTA server management has been included in delta files.
+    - Testsuites for OS upgrade have been developed.
+    - Delta file generators have been improved.
+  - Open Source
+    - CVE-2021-33910 patch has been applied to systemd.
+
+
+### System (Base)
+
+#### New and changed features
+
+- Open Source
+  - ccache
+    - Version 3.1.6 has been upgraded to version 4.4.
+  - libxml2
+    - Version 2.9.10 has been upgraded to version 2.9.12.
+  - gtest
+    - Version 1.8.0 has been upgraded to version 1.10.0.
+- System-settings
+  - Image validation checker has been removed about two keys.

```

Comment:
Would this be more accurate?
```suggestion
  - Two keys for Image validation checker has been removed.

```

Reply:
> Would this be more accurate?

No, below keys still exist, now.
I just want to express that Image Validation checker removed when two keys is being used.